cfme/tests/physical_infrastructure/ui/test_physical_overview.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from cfme.physical.overview import PhysicalOverview
+from cfme.physical.provider.lenovo import LenovoProvider
+from cfme.utils.appliance.implementations.ui import navigate_to
+
+pytestmark = [pytest.mark.tier(3),
+    pytest.mark.provider([LenovoProvider])]
+
+
+def test_physical_overview_page():
+    view = navigate_to(PhysicalOverview, 'All')
+    assert view.is_displayed
+
+
+def test_physical_overview_servers_number(appliance, provider):
+    view = navigate_to(PhysicalOverview, 'All')
+    servers = appliance.collections.physical_servers.all(provider)
+    assert view.servers.value == len(servers)
+
+
+def test_physical_overview_switches_number(appliance):
+    view = navigate_to(PhysicalOverview, 'All')
```

Comment:
Same here as above - fetch the all() count first and then navigate and use the view from navigation second.

Or better yet - query wrapanapi first, and view second :)

Reply:
Hey @mshriver, thanks for your review. As far as I know wrapanapi fetches information from only 1 provider, and Overview page informs counts servers/switches/racks... from all physical providers connected to MIQ. What would be a better way to handle this issue?

Reply:
Also, fetch switches from wrapanapi depends on [wrapanapi/294](https://github.com/ManageIQ/wrapanapi/pull/294)